types/wlr_xdg_output.c

```diff
@@ -6,6 +6,7 @@
 #include <wlr/types/wlr_xdg_output.h>
 #include <wlr/util/log.h>
 #include "xdg-output-unstable-v1-protocol.h"
+#include "types/wlr_output_layout.h"
```

Comment:
Hmmm. I'm not a fan of this.

Reply:
Yeah, this interface can probably just work with the `wlr_output`. `wlr_cursor.c` uses it too.